third_party/blink/public/platform/web_source_buffer.h

```diff
@@ -140,6 +143,13 @@ class WebSourceBuffer {
   // After this method is called, this WebSourceBuffer should never use the
   // client pointer passed to SetClient().
   virtual void RemovedFromMediaSource() = 0;
+
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+  // Return the highest presentation timestamp written to SbPlayer.
+  virtual double write_head(ExceptionState& exception_state) const {
```

Comment:
nit: align the naming with other functions, like `WriteHead` in web_source_buffer.h.

Reply:
And we should consider to allow this function to `=0`, so this is not necessary to be implemented.